surveys/2024-annual-survey/questions.md

```diff
@@ -509,8 +509,8 @@ Activities:
 - Write, comment on, contribute to discussion of, or provide edits to an open RFC
 - Discuss the Rust project in an official chat or forum (internals.rust-lang.org, Rust Zulip etc.)
 - Open an issue on any repo in the rust-lang GitHub organization
-- Contribute code changes (including tests) to any project in the rust-lang GitHub organization
-- Contribute non-code changes (documentation, comments, etc.) to any project in the rust-lang GitHub organization
+- Contribute code changes (including tests) to any open-source Rust project
+- Contribute non-code changes (documentation, comments, etc.) to any open-source Rust project
```

Comment:
```suggestion
- Contribute code changes (including tests) to any free and Rust open source project
- Contribute non-code changes (documentation, comments, etc.) to any Rust free and open source project
```

I'm happy with extending to Rust projects outside our orga. Would be even happier with a wording that includes also *libre* software (without using that word which is less widespread). Or is it too much bikeshedding?

Reply:
There is no such thing as "too much bikeshedding" when it comes to the survey :laughing: Since we're asking about community interactions, the important thing is the "open-source" part, I think, i.e. it doesn't really matter what kind of software it is, as long as it's open-source, it can be classified as a community interaction, IMO.